doc/man3/EVP_EncryptInit.pod

```diff
@@ -243,6 +254,12 @@ IDENTIFIER as such it ignores the cipher parameters and 40 bit RC2 and
 identifier or does not have ASN1 support this function will return
 B<NID_undef>.
 
+EVP_CIPHER_name() and EVP_CIPHER_CTX_name() return the name of the passed
+cipher or context.
+
+EVP_CIPHER_provider() returns a B<OSSL_PROVIDER> pointer to the provider
```

Comment:
I'm still seeing _a_ here not _an_.
Dunno why.

Reply:
I did change it in EVP_DigestInit.pod....  what, that's not enough????? :wink: